scripts/seed-database.ts

```diff
@@ -0,0 +1,81 @@
+import { createConnection, Connection } from 'mysql2/promise'
+import { scryptHash } from '../src/plugins/custom/scrypt.js'
+
+async function seed () {
+  const connection: Connection = await createConnection({
+    multipleStatements: true,
+    host: process.env.MYSQL_HOST,
+    port: Number(process.env.MYSQL_PORT),
+    database: process.env.MYSQL_DATABASE,
+    user: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD
+  })
+
+  try {
+    await truncateTables(connection)
+    await seedUsers(connection)
+  } catch (error) {
+    console.error('Error seeding database:', error)
+  } finally {
+    await connection.end()
+  }
+}
+
+async function truncateTables (connection: Connection) {
+  const [tables]: any[] = await connection.query('SHOW TABLES')
+
+  if (tables.length > 0) {
```

Comment:
```suggestion
if (tables.length) {
```

Reply:
You would perform a nullish check, although it should work here, this is semantically wrong imho.